jx_sqlite/query_table.py

```diff
@@ -288,7 +288,12 @@ def query(self, query):
                     data = Data()
                     for c in index_to_columns.values():
                         if c.push_child == ".":
-                            data[c.push_name] = c.pull(result.data[0])
+                            if not data[c.push_name]:
```

Comment:
there is a `else` clause that should have the same logic for the sake of symmetry.